### DISCUSSION TITLE
Is it possible to outline the focused application?

### DISCUSSION THREAD

**Discussion by matdave** [asmvik/yabai#2411] [Q&A] [answered]
I see there is an option to set the opacity of unfocused applications, but I was wondering if there is a way to add an outline around the active/inactive applications. This is a fairly common option in auto-tilers, so not sure if it's been implemented or if people have a solution.

**Comment by Bellavene** [comment, marked as answer]
Strange that you didn't find it out already : )
https://github.com/FelixKratz/JankyBorders

**Reply by matdave** [reply]
Thanks! This is what I was looking for!
